### PR TITLE
Feature/fix varnish docker

### PR DESCRIPTION
**Pull request by Simperfit**

### CODE REVIEW

**Comment by Simperfit** [comment]
@dunglas

File: docker/nginx/conf.d/default.conf [hunk 1]
```diff
@@ -26,8 +24,7 @@ server {
     }
     # PROD
     location ~ ^/app\.php(/|$) {
-        set $upstream_host app;
```
Comment (dunglas) [inline docker/nginx/conf.d/default.conf:29]: This should be reverted to allow dynamic resolution.

File: web/config.php [hunk 2]
```diff
@@ -14,10 +14,10 @@
     exit("This script cannot be run from the CLI. Run it from a browser.\n");
 }
 
-if (!in_array(@$_SERVER['REMOTE_ADDR'], [
```
Comment (dunglas) [inline web/config.php:17]: This should be reverted too.

**Review by dunglas — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -37,8 +37,8 @@ services:
       # - ./docker/db/data:/var/lib/mysql:rw
   nginx:
     build:
-      context: ./docker/nginx
-      dockerfile: ./Dockerfile
+      context: ./
+      dockerfile: ./Dockerfile.nginx
```
Comment (dunglas) [inline docker-compose.yml:41]: If we switch to this convention, shouldn't we do it for all Dockerfiles (for consistency)?

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by dunglas** [comment]
ping @teohhanhui

File: docker/nginx/conf.d/default.conf (hunk 1 above)
Comment (Simperfit) [reply docker/nginx/conf.d/default.conf:29]: It does not work with dynamic resolution

File: web/config.php (hunk 2 above)
Comment (Simperfit) [reply web/config.php:17]: yep

File: docker-compose.yml (hunk 3 above)
Comment (Simperfit) [reply docker-compose.yml:41]: I don't think so, we need this one in order to do the ADD, we don't need all Dockerfile to have the app context

File: docker/nginx/conf.d/default.conf (hunk 1 above)
Comment (dunglas) [reply docker/nginx/conf.d/default.conf:29]: I've just checked and it works, this must be reverted.

**Pull request closed without merging**